README.md

```diff
@@ -1,5 +1,6 @@
 # puppet-ferm
 
+[![Puppet Forge modules by Vox Pupuli](https://img.shields.io/puppetforge/mc/puppet.svg)](https://forge.puppetlabs.com/puppet)
```

Comment:
This says very little for this particular module. It says something about the VP org.